dspace/config/dspace.cfg

```diff
@@ -1966,7 +1966,8 @@ mail.helpdesk = ${mail.admin}
 # Should all Request Copy emails go to the helpdesk instead of the item submitter?
 request.item.helpdesk.override = false
 
-
+# URL of ORCID API
+orcid.api.url = https://api.orcid.org/v2.0
```

Comment:
I believe this is the source of the error that @kshepherd noted.  This should be set to https://pub.orcid.org/v2.0 by default, as that's the *public* API.  The api.orcid.org path is for Members only, and requires authentication.